<launch>

    <node name="color_mask" pkg="robot_arm" type="color_mask.py" 
          output="screen" respawn="true"/>

    <node name="detect_sherds" pkg="robot_arm" type="detect_sherds.py" 
          output="screen" respawn="true"/>

    <node name="read_scale" pkg="robot_arm" type="read_scale.py"
	  output="screen" respawn="true"/>

</launch>
